clar2wasm/src/words/conditionals.rs

```diff
@@ -1173,8 +1175,17 @@ mod tests {
         crosscheck("(asserts! true (err u1))", Ok(Some(Value::Bool(true))));
     }
 
+    #[ignore = "see issue: #385"]
```

Comment:
This issue should be merged with the current one. This is part of supporting runtime errors.

Reply:
Okay, let's postpone this.